specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/integrations.json

```diff
@@ -0,0 +1,737 @@
+        "description": "Creates or updates an integration over a given scope",
+        "operationId": "Integrations_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/IntegrationId"
+          },
+          {
+            "$ref": "#/parameters/IntegrationBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Updated",
+            "schema": {
+              "$ref": "#/definitions/Integration"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/Integration"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete an integration over subscription scope": {
+            "$ref": "./examples/Integrations/DeleteIntegration_example.json"
+          },
+          "Delete an integration over security connector scope": {
+            "$ref": "./examples/Integrations/DeleteSecurityConnectorIntegration_example.json"
+          },
+          "Delete an integration over management group scope": {
+            "$ref": "./examples/Integrations/DeleteManagementGroupIntegration_example.json"
+          }
+        },
+        "tags": [
+          "Integrations"
+        ],
+        "description": "Delete an integration over a given scope",
+        "operationId": "Integrations_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/IntegrationId"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted - The operation still in progress",
+            "headers": {
+              "location": {
+                "type": "string",
+                "description": "Location URL for the deletion status"
+              }
+            }
+          },
+          "204": {
+            "description": "No Content - Integration does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Security/integrations/{integrationId}/users": {
```

Comment:
/{scope}/providers/Microsoft.Security/integrations/{integrationId}/users is not a valid ARM resource ID - it looks like a mixed use of nested/child resource and an extension resource - I believe your intention was to create a nested resource type - please check how Microsoft.Network/networkSecurityGroups and Microsoft.Network/networkSecurityGroups/securityRules behaves - this might be what you wanted